drcp-format/README.md

```diff
@@ -28,6 +28,9 @@ The individual components:
   - `c:<constraint tag>`: _Optional_. A hint which constraint triggered the inference.
   - `l:<filtering algorithm>`: _Optional_. A hint which filtering algorithm identified the inference.
 
+  If there is no `<propagated>`, then the inference reads that the premises imply false.
```

Comment:
Just a clarifying question; this would mean that this propagation led to an empty domain? Or are there other scenarios in which this would occur?